packages/eas-cli/src/graphql/types/Actor.ts

```diff
@@ -0,0 +1,11 @@
+import { AccessToken } from './AccessToken';
+import { Account } from './Account';
+
+export interface Actor {
```

Comment:
Hmmm, maybe this could be a class instead of an interface. This way we could have both type checking and easy access to the class name (and then we wouldn't need the first argument for QueryBuilder). See how `@expo/entity` does a similar thing with entity classes.

Reply:
Feel free to change that in a follow-up PR.